dnf5.spec

```diff
@@ -247,15 +251,19 @@ License:        LGPL-2.1-or-later
 Requires:       libsolv%{?_isa} >= %{libsolv_version}
 Requires:       librepo%{?_isa} >= %{librepo_version}
 Requires:       sqlite-libs%{?_isa} >= %{sqlite_version}
+%if 0%{?fedora} > 38
 Conflicts:      dnf-data < 4.16.0
+%endif
 
 %description -n libdnf5
 Package management library.
 
 %files -n libdnf5
+%if 0%{?fedora} > 38
```

Comment:
Here we would need to do the changes also in `dnf` spec I guess as this is not already being provided there...